### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by dszmigielski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
-# Splunk Distribution of OpenTelemetry .NET
+# Splunk Distribution of OpenTelemetry Go
```
Comment (Kielek) [inline README.md:1]:
```suggestion
# Splunk Distribution of OpenTelemetry .NET
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,11 @@
...
+
+[![OpenTelemetry Dotnet](https://img.shields.io/badge/OTel-1.3.1-blueviolet)](https://github.com/open-telemetry/opentelemetry-dotnet/releases/tag/core-1.3.1)
```
Comment (Kielek) [inline README.md:3]: I think that we should have a link to AutoInstrumentation instead of the OTel .NET (or both of them).

File: docs/README.md [hunk 3]
```diff
@@ -19,13 +15,41 @@ This Splunk distribution comes with the following defaults:
 - OTLP over HTTP exporter configured to send spans to a locally running [Splunk OpenTelemetry
   Collector](https://github.com/signalfx/splunk-otel-collector)
 
-Install the distribution:
+## Get started
```
Comment (Kielek) [inline docs/README.md:18]: We have Get started and Getting started.

**Review by Kielek — APPROVED** [review]

**Review by Kielek — APPROVED** [review]

File: docs/README.md [hunk 4]
```diff
@@ -19,13 +15,39 @@ This Splunk distribution comes with the following defaults:
...
 
-TODO
+> The path where you put the binaries is referenced as `$INSTALL_DIR`
 
 ### Basic configuration
 
-TODO
+When running your application, make sure to:
+
+1. Set the [resources](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/v0.3.1-beta.1/docs/config.md#resources).
```
Comment (Kielek) [inline docs/README.md:29]: I think that you are missing info about our plugins. I know that both are doing nothing, but it should be documented.

**Review by Kielek — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,5 +1,13 @@
 # Splunk Distribution of OpenTelemetry .NET
 
+[![OpenTelemetry .NET](https://img.shields.io/badge/OTel-1.3.1-blueviolet)](https://github.com/open-telemetry/opentelemetry-dotnet/releases/tag/core-1.3.1)
+[![OpenTelemetry .NET Auto Instrumentation](https://img.shields.io/badge/OTel-v0.3.1-blueviolet)](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/tag/v0.3.1-beta.1)
+[![Splunk GDI Specification](https://img.shields.io/badge/GDI-1.3.0-blueviolet)](https://github.com/signalfx/gdi-specification/releases/tag/v1.3.0)
+[![Keep a Changelog](https://img.shields.io/badge/changelog-Keep%20a%20Changelog-%23E05735)](CHANGELOG.md)
```
Comment (Kielek) [inline README.md:6]: Please check following screen: 
image

File: docs/README.md [hunk 6]
```diff
@@ -19,13 +15,39 @@ This Splunk distribution comes with the following defaults:
 - OTLP over HTTP exporter configured to send spans to a locally running [Splunk OpenTelemetry
   Collector](https://github.com/signalfx/splunk-otel-collector)
 
-Install the distribution:
+### Install
+
+Download and extract the appropriate binaries from
+[the latest release](https://github.com/signalfx/splunk-otel-dotnet/releases/latest).
```
Comment (theletterf) [inline docs/README.md:21]:
```suggestion
Download and install the latest binaries from
[the latest release](https://github.com/signalfx/splunk-otel-dotnet/releases/latest).
```

File: docs/README.md [hunk 7]
```diff
@@ -19,13 +15,39 @@ This Splunk distribution comes with the following defaults:
...
 
-TODO
+> The path where you put the binaries is referenced as `$INSTALL_DIR`
```
Comment (theletterf) [inline docs/README.md:23]:
```suggestion
> The path where you place the binaries is referenced as `$INSTALL_DIR` in this documentation.
```

File: docs/advanced-config.md [hunk 8]
```diff
@@ -1 +1,8 @@
 # Advanced configuration
+
+## OpenTelemetry configuration
+
+Please see [Open Telemetry Auto Instrumentation documentation](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/v0.3.1-beta.1/docs/config.md)
```
Comment (theletterf) [inline docs/advanced-config.md:5]:
```suggestion
See [Open Telemetry Auto Instrumentation documentation](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/v0.3.1-beta.1/docs/config.md)
```

File: docs/README.md (hunk 4 above)
Comment (dszmigielski) [reply docs/README.md:29]: How do you want to go about it? "Here are some plugins that do nothing"? 🤔

File: README.md [hunk 9]
```diff
@@ -1,5 +1,13 @@
...
+[![LICENSE](https://img.shields.io/github/license/signalfx/splunk-otel-dotent)](LICENSE.md)
```
Comment (Kielek) [inline README.md:7]:
```suggestion
[![LICENSE](https://img.shields.io/github/license/signalfx/splunk-otel-dotnet)](LICENSE)
```

**Pull request merged**